pkg/cli/config/config.go

```diff
@@ -85,6 +98,21 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	return bpConfig, nil
 }
 
+// GetBackplaneURL returns API URL
+func (config *BackplaneConfiguration) GetBackplaneURL() (string, error) {
+
+	ocmEnv, err := ocm.DefaultOCMInterface.GetOCMEnvironment()
+	if err != nil {
+		return "", err
+	}
+	url, ok := ocmEnv.GetBackplaneURL()
+	if !ok {
+		return "", fmt.Errorf("no backplane API defined for %v", ocmEnv.Name())
+	}
+	logger.Infof("backplane URL fetch via OCM environment: %s", url)
```

Comment:
Backplane is a noun, so let's abide by that :)
```suggestion
	logger.Infof("Backplane URL retrieved via OCM environment: %s", url)
```